files/zh-cn/mdn/community/issues/index.md

```diff
@@ -0,0 +1,216 @@
+分类的整个过程包括一些[一般](#一般分类任务)和一些[议题特定任务](#议题特定分类任务)。
+
+### 一般分类任务
+
+- 在议题被创建后，其会被自动设置 `needs triage` 标签。你可以搜索该标签来查找需要[进行分类](#议题特定分类任务)的议题。在议题被分类之前，贡献者或其他任何人都不应该处理该议题。（分类人员请记得在对议题进行分类后删除 `needs triage` 标签。）
+- 在 [mdn/content 仓库](https://github.com/mdn/content/issues)中，还会自动设置额外的 `Content:` 标签，例如 `Content:CSS` 或 `Content:WebAPI`。这是根据议题中提到的 MDN URL 设置的。你可以使用特定内容的标签来查找需要在你的特定主题领域中进行分类的议题。
+- 如果议题涉及的活动并不是 en-US 区域的，请设置适当的标签（例如，`l10n-fr`、`l10n-zh` 或 `l10n-ja`）。这些区域的团队将会接手这些议题并对其进行分类。
+- 你不需要一直主动对议题进行分类。每周抽出一些时间（例如 30 分钟）定期对你负责的议题进行分类。分类不一定要作为同步会议的一部分进行，甚至不需要与其他人同时进行，但应定期进行，以确保未分类错误的积压不会太多。
+- 除了每周对新增的议题进行分类外，还应该查看旧议题列表，以查看是否有任何议题处于停滞状态、需要关闭或不再相关。`idle` 标签会自动设置到 30 天内没有活动的议题上。
+  - 检查仍处于打开状态的已分配议题，以检查受理人（assignee）是否正在进行处理。如果受理人在被分配后一周内没有任何进展，请询问他们是否仍有时间来解决该议题。如果又过了一周而没有任何进展，请取消分配并留下评论，说明你可以将该议题提供给其他感兴趣的贡献者。
+  - 如果已创建用于解决议题的拉取请求，但是在一周内未得到审员，请有礼貌地提醒审查员，询问他们是否可以处理该请求。
+  - 如果修复议题的拉取请求的审查意见在一周内未被解决，请询问作者是否可以回应他们的审查意见。如果又过了一周，且你有时间，请自行处理审查意见，或者关闭拉取请求并取消相关议题的分配。
+
+### 议题特定分类任务
+
+这是在对每个议题进行分类时应遵循的指南。
+
+#### 检查议题是否有效
+
+在审核议题的有效性时，请记住以下几点：
+
+- 检查提出的议题是否有效，以及修复是否会改善为读者和网站提供的内容。
+- 评估修复的影响是较小的还是整个网站的。
+- 评估议题的修复是否需要讨论，如果需要，请让作者先发起[讨论](https://github.com/orgs/mdn/discussions)。
+- 检查议题是否符合我们的[写作指南](/zh-CN/docs/MDN/Writing_guidelines/Writing_style_guide)和[模板](/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Page_types)。
+- 检查添加链接的建议是否符合我们的[外部链接政策](/zh-CN/docs/MDN/Writing_guidelines/Writing_style_guide#外部链接)。
+
+#### 检查议题信息的完整性
+
+请根据以下清单检查每个议题，以确保议题包含了描述信息，以便有人可以开始处理该错误：
+
+- 存在问题的 MDN Web 文档页面的 URL，或者如果问题存在于多个页面上，则为示例 MDN Web 文档页面的 URL。
+- 发现问题的 MDN Web 文档页面的特定标题或部分
+- 对不正确、无用、不完整或缺失信息的清晰描述
+
+如果上述的任意信息不存在，那么你应该要求议题的作者提供这些细节，并为议题添加 `needs info` 标签。只有在提供了这些细节后（之后，你可以删除 `needs info` 标签），才能继续对议题的分类。可以用一周的时间等待作者回复。
+
+#### 设置优先级标签
+
+请根据议题的严重程度为每一个错误设置优先级标签，以帮助那些想要解决最重要的议题或领域的人。
+
+- 严重议题：此类议题需要尽快修复，无论它出现在网站的哪个位置。此类议题可能会严重损害 MDN 的声誉和/或损害用户。此类议题的示例包括错误的代码片段，如果在生产中使用，可能会产生严重的安全问题和不受欢迎的内容，例如恶意软件、亵渎、色情、仇恨言论或指向此类内容的链接。
+
+  - 标签：`p0`（将立即得到解决）
+
+- 主要议题：此类议题可能会严重影响页面的实用性。例如，大量过时的信息、不起作用的复杂而重要的代码示例、大量糟糕且难以理解的行文，或大量损坏的链接。
+
+  - 标签：`p1`（将很快得到解决）和 `p2`（将很快得到解决，但优先级更改的项目将优先得到解决）
+
+- 小议题：这是一种可以使现有内容更好但不影响学习或仅对学习产生轻微影响的改进议题。由于这些类型的议题不是主动计划的，因此欢迎并非常感谢贡献者的帮助来解决这些议题。解决其中一些议题也可以为刚开始熟悉贡献流程的初学者提供必要的练习。示例包括拼写错误、语法错误、损坏的链接、少量过时的信息或糟糕的行文，或者不起作用的代码片段。
+
+  - 标签：`p3`（无法预期问题何时可以得到解决）
+
+一般来说，严重议题应立即修复，最有可能由 MDN Web 文档的工作人员和同行进行处理。
+
+#### 添加有用的信息
+
+如果可能，请添加可以帮助贡献者解决议题的信息。这些信息可以是步骤、一般方法、指向其他类似已解决议题的链接或阅读资源。对于标记为 `good first issue` 的议题，则特别需要一个良好的计划或步骤，这可以帮助新贡献者快速上手。你可以将此任务的时间限制在 5-10 分钟内。
+
+例如，作为分类人员，你可以将以下信息添加到你正在分类的议题中：
+
+```md
+对于修复此议题的人来说，似乎需要以下内容：
+
+- 更新标题甲下的第一段，以更正乙的问题
+- 添加一段关于丙的描述
+- 更新链接甲中的兼容性数据
+```
+
+#### 设置其他标签
+
+接下来，请根据需要设置以下标签：
+
+- `effort: small`、`effort: medium`、`effort: large`：一些贡献者喜欢根据修复错误所需的时间和精力来搜索错误。因此，如果可能的话，你应该尝试提供所需精力的估计。
+- `good first issue`：如果议题的修复非常简单，并且修复议题将为刚开始熟悉贡献流程的初学者提供良好的练习，请在议题上设置此标签。
+- `help wanted`：如果议题需要来自了解或熟悉该主题的人的帮助，请设置此标签。这是一个流行的标签，一些贡献者使用它来搜索他们熟悉或精通的开源项目中的议题。
+- `broken link internal`、`broken link external`：如果议题涉及到指向内部或外部页面的损坏链接，请设置适当的标签。
```

Comment:
Currently there is only `broken link external` label in mdn-content repo, should we remove mention of `broken link internal` 🤔 

![image](https://github.com/mdn/translated-content/assets/40999116/d8c125b3-bd69-4b40-a57c-4eea0dfdc92d)

Reply:
the en-US content is [here](https://github.com/mdn/content/blob/7eebbf7990a48b38b1094c9ea9b6607969b34cd9/files/en-us/mdn/community/issues/index.md?plain=1#L222)

@bsmth what's your opinion on this?